### PR TITLE
feat(bundler-cli,bundler-core): improved cli input, output and verification

### PR DESCRIPTION
**Pull request by melancholiai**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✔                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✔                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✖                                                                       |

### CODE REVIEW

File: packages/cli/src/commands/verify/util.ts [hunk 1]
```diff
@@ -1,3 +1,10 @@
+import { homedir } from 'os';
+import { readFile } from 'fs/promises';
+import { join } from 'path';
+import { decode } from 'jsonwebtoken';
+
+const MILLISECONDS_IN_SECOND = 1000;
```
Comment (syncush) [inline packages/cli/src/commands/verify/util.ts:6]: Shouldn't it be nested under common/constants?

File: packages/core/package.json [hunk 2]
```diff
@@ -28,7 +28,7 @@
   ],
   "dependencies": {
     "@map-colonies/bundler-child-process": "^1.0.3",
-    "@map-colonies/bundler-common": "^1.0.3",
+    "@map-colonies/bundler-common": "*",
```
Comment (syncush) [inline packages/core/package.json:31]: once we publish we will set the correct ver, right?

File: packages/github/src/client/interfaces.ts [hunk 3]
```diff
@@ -1,3 +1,4 @@
+import { RepositoryId } from '@map-colonies/bundler-common/src';
```
Comment (syncush) [inline packages/github/src/client/interfaces.ts]: why are we specifying /src?

**Review by syncush — CHANGES_REQUESTED** [review]

File: packages/cli/src/commands/verify/util.ts (hunk 1 above)
Comment (melancholiai) [reply packages/cli/src/commands/verify/util.ts:6]: this variable is being used here and here only

File: packages/core/package.json (hunk 2 above)
Comment (melancholiai) [reply packages/core/package.json:31]: true

File: packages/github/src/client/interfaces.ts (hunk 3 above)
Comment (melancholiai) [reply packages/github/src/client/interfaces.ts]: fixed.

**Review by syncush — APPROVED** [review]

**Pull request merged**